ts/Accessibility/Accessibility.ts

```diff
@@ -469,6 +471,9 @@ namespace Accessibility {
                 if (a11y && !a11y.zombie) {
                     a11y.update();
                 }
+                // If a11y has been enabled = false, and is now enabled = true
+                svg.removeAttribute('role');
+                this.wasAccessibilityEnabled = true;
```

Comment:
Do we need this prop? Could just always remove the role attr if a11y is enabled, and add it if not?